RadixWallet/Clients/KeychainClient/KeychainHolder.swift

```diff
@@ -38,6 +38,28 @@ extension KeychainHolder {
 	) throws -> Bool {
 		try keychain.contains(key.rawValue.rawValue, withoutAuthenticationUI: !showAuthPrompt)
 	}
+
+	func getAllKeysMatching(
+		synchronizable needleIsSynchronizable: Bool? = false,
+		accessibility needleAccessibility: KeychainAccess.Accessibility? = .whenPasscodeSetThisDeviceOnly
+	) -> [String] {
+		keychain.allItems()
+			.filter {
+				if let isSynchronizable = $0["synchronizable"] as? Bool, let needle = needleIsSynchronizable {
```

Comment:
keys: "synchronizable", "accessibility" and "key" are keys internal to Keychain.